packages/docs/src/styles/components/pageHeader.scss

```diff
@@ -17,3 +17,10 @@
   margin-right: $spacer-half;
   vertical-align: middle;
 }
+
+#header-full-width {
```

Comment:
I thought we avoided styling `id`s - because an `id` is usually reserved for JS interactivity and not CSS. Not sure if that's a hard rule though.

Reply:
Oh didn't realize, I'll just make these classes.